client-v2/src/shared/components/input/InputImage.tsx

```diff
@@ -93,6 +125,36 @@ class InputImage extends React.Component<ComponentProps, ComponentState> {
         console.log('@todo:handle error', err);
       });
   };
+
+  public handlePasteImgSrcInputChange = (
+    e: React.FormEvent<HTMLInputElement>
+  ) => {
+    e.preventDefault();
+    this.setState({ imageSrc: e.currentTarget.value });
+  };
+
+  public handlePasteImgSrcInputSubmit = (e: React.KeyboardEvent) => {
+    events.imageAdded(this.props.frontId, 'paste');
+    e.persist();
+    if (e.keyCode === 13) {
```

Comment:
could we use a variable here to avoid the magic number?

Reply:
Good shout. I'm amending to: 
```
  public handlePasteImgSrcSubmit = (keyCode: number) => (
    e: React.KeyboardEvent
  ) => {
    events.imageAdded(this.props.frontId, 'paste');
    e.persist();
    if (e.keyCode === keyCode) {
```